src/ui/widgets/EmbeddedDisplay/embeddedDisplay.tsx

```diff
@@ -209,17 +216,21 @@ export const EmbeddedDisplay = (
 
   if (props.border?.style === BorderStyle.GroupBox) {
     return (
-      <MacroContext.Provider value={embeddedDisplayMacroContext}>
-        <GroupBoxComponent name={resolvedName} styleOpt={0}>
-          <ThemeProvider theme={diamondTheme}>{component}</ThemeProvider>
-        </GroupBoxComponent>
-      </MacroContext.Provider>
+      <ThemeProvider theme={props.theme ?? theme}>
```

Comment:
Rather than using the default MUI theme here if the client application doesn't pass one in, we should use a "default" theme we have created that has Phoebus defaults. So the diamondTheme you previously created (which contains Phoebus defaults for font and colours etc) could be used (but perhaps renamed to `phoebusTheme` or `defaultTheme`).